src/python/WMCore/FwkJobReport/Report.py

```diff
@@ -1233,9 +1238,10 @@ def getAllFileRefs(self):
 
         fileRefs = []
         for step in self.data.steps:
-            tmpRefs = self.getAllFileRefsFromStep(step=step)
-            if len(tmpRefs) > 0:
-                fileRefs.extend(tmpRefs)
+            fileDict = self.getAllFileRefsFromStep(step=step)
+            for tmpRefs in fileDict.values():
+                if len(tmpRefs) > 0:
```

Comment:
No, this is needed since self.getAllFileRefsFromStep returns a dict now, not a list.